.github/workflows/_bazel-build-test.yml

```diff
@@ -11,13 +11,25 @@ on:
         required: true
         type: string
         description: Name of the base docker image to build with.
+      cuda-version:
```

Comment:
I guess you find the `cuda-version` param from windows CI, that's probably ok I think